db/migrate/20220830134159_create_taxes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTaxes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :taxes do |t|
+      t.string :name
+      t.text :description
+      t.date :from_date
+      t.date :until_date
+      t.references :car, null: false, foreign_key: true
+
```

Comment:
amount filed is missing

Reply:
Well, the amount should be an integer because storing decimals isn't very precise and can lead to data loss. So the convention is to store money-related info in cents. I was planning on creating another task to agg gem money and to deal with all the money-related amounts in order to not overload current tasks

Reply:
I think I manage to implement money-rails gem in table of chargings and in the model , that could be seen in  create_chargings branch.

Reply:
Yes, @galixa7 please take a look at Stoyan's PR https://github.com/Sf-60-Ruby-Cisco/BE-Ruby-Project/pull/33#pullrequestreview-1093716139